src/webhooks/pubsub/slackNotifications.ts

```diff
@@ -235,58 +229,41 @@ export const constructSlackMessage = (
                 hoursLeft === 1 ? `${hoursLeft} hour` : `${hoursLeft} hours`;
               actFastIssues.push({
                 triageBy,
-                fields: [
-                  {
-                    text: `${
-                      actFastIssues.length + 1
-                    }. <${url}|#${number} ${escapedIssueTitle}>`,
-                    type: 'mrkdwn',
-                  },
-                  { text: `${hoursText} ${minutesText} left`, type: 'mrkdwn' },
-                ],
+                issueLink: `<${url}|#${number} ${escapedIssueTitle}>`,
+                timeLeft: `${hoursText} ${minutesText} left`,
               });
             } else {
               if (daysLeft < 1) {
                 triageQueueIssues.push({
                   triageBy,
-                  fields: [
-                    {
-                      text: `${
-                        triageQueueIssues.length + 1
-                      }. <${url}|#${number} ${escapedIssueTitle}>`,
-                      type: 'mrkdwn',
-                    },
-                    { text: `${hoursLeft} hours left`, type: 'mrkdwn' },
-                  ],
+                  issueLink: `<${url}|#${number} ${escapedIssueTitle}>`,
+                  timeLeft: `${hoursLeft} hours left`,
                 });
               } else {
                 const daysText =
                   daysLeft === 1 ? `${daysLeft} day` : `${daysLeft} days`;
                 triageQueueIssues.push({
                   triageBy,
-                  fields: [
-                    {
-                      text: `${
-                        triageQueueIssues.length + 1
-                      }. <${url}|#${number} ${escapedIssueTitle}>`,
-                      type: 'mrkdwn',
-                    },
-                    { text: `${daysText} left`, type: 'mrkdwn' },
-                  ],
+                  issueLink: `<${url}|#${number} ${escapedIssueTitle}>`,
+                  timeLeft: `${daysText} left`,
                 });
               }
             }
           }
         );
       });
+  
       const sortAndFlattenIssuesArray = (issues) =>
         issues
           .sort(
             (a, b) =>
               moment(a.triageBy).valueOf() - moment(b.triageBy).valueOf()
           )
-          .map((item) => item.fields)
+          .map((item, index) => {
+            return addOrderingToSlackMessageItem(item, index + 1).fields;
```

Comment:
Wondering what this behavior would look like if there are issues in both the overdue and act fast queue. Will it reset the numbering for each respective queue?

Rough Example:

Overdue 
1. Issue One
2. Issue Two

Act Fast
1. Issue Three
2. Issue Four

Reply:
I've added another test that covers this scenario in the latest patcheset.